src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -57,7 +73,7 @@ internal void PopulateAutoCompleteCandidates()
         /// The search elements can be made to appear in the node autocomplete search dialog.
         /// </summary>
         /// <returns>collection of node search elements</returns>
-        internal IEnumerable<NodeSearchElement> GetMatchingNodes()
+        internal IEnumerable<NodeSearchElement> GetMatchingSearchElements()
```

Comment:
Any reason for this change?

Reply:
Renamed since the function itself does not really return nodes but rather than search elements, just to reduce confusion in the future